source/languageHandler.py

```diff
@@ -151,6 +151,7 @@ def setLanguage(lang):
 	try:
 		if lang=="Windows":
 			localeName=getWindowsLanguage()
+			locale.setlocale(locale.LC_ALL, localeName)

```

Comment:
Rather than adding this line, I think it would be better to move the existing block that tries setlocale two different ways, down below setThreadLocale, but outdented by one tab (I.e. outside of the if lang == 'windows' check). So everything from localeChanged = False, up to but not including #Set the windows locale for this thread...
Then that code will run whether or not lang is 'windows'.

Reply:
Makes sense, I've moved it out now